LongevityWorldCup.Website/wwwroot/partials/leaderboard-content.html

```diff
@@ -2395,8 +2430,69 @@ <h3>Proofs</h3>
             const biomarkerValues = prepareBiomarkerValues(entry, chronoAgeAtEntry);
             const phenoAge = window.PhenoAge.calculatePhenoAge(biomarkerValues);
             phenoAges.push(phenoAge.toFixed(1));
+
+            const now = new Date();
+            const oneYearAgo = new Date();
+            oneYearAgo.setFullYear(now.getFullYear() - 2);
+            const isEntryFromThisSeason = entryDate >= currentSeasonStartDate && entryDate <= now;
+            const isEntryFromTheLastYear = entryDate >= oneYearAgo && entryDate <= now;
```

Comment:
This is currently the last two years (instead of last year, as the names suggest)